modules/terraform-aci-contract/variables.tf

```diff
@@ -155,4 +238,20 @@ variable "subjects" {
     ]))
     error_message = "`priority`: Allowed values are `default`, `level1`, `level2` or `level3`."
   }
-}
+
+  validation {
+    condition = alltrue(flatten([
+      for s in var.subjects : [for f in coalesce(s.provider_to_consumer_filters, []) : f.priority == null || try(contains(["default", "level1", "level2", "level3"], f.priority), false)]
+    ]))
+    error_message = "`priority`: Allowed values are `default`, `level1`, `level2` or `level3`."
+  }
```

Comment:
Missing `action` validation for `provider_to_consumer_filters`. Also missing `action` and `priority` validations for `consumer_to_provider_filters`.